addons/portal/controllers/mail.py

```diff
@@ -265,3 +265,8 @@ def _redirect_to_record(cls, model, res_id, access_token=None, **kwargs):
                             url = url.replace(query=urls.url_encode(url_params)).to_url()
                         return request.redirect(url)
         return super(MailController, cls)._redirect_to_record(model, res_id, access_token=access_token, **kwargs)
+
+    # Add website=True to support the portal layout
```

Comment:
website=True is supported in portal :thinking: ? can't we simply change the layout ?

Reply:
Ok then ^^